chia/wallet/wallet_action_store.py

```diff
@@ -37,11 +37,11 @@ async def create(cls, db_wrapper: DBWrapper):
             )
         )
 
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS name on action_queue(name)")
+        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS aq_name on action_queue(name)")
```

Comment:
why do you rename this?
It means everyone will need to spend a bunch of time at startup (when upgrading) to create a new index.

Reply:
All wallet dbs synced on my side are missing indexes on action_queue table.
Reason is, that the index name "name" was used multiple times for different tables in the scripts.
For instance, the here mentioned index "name" is on my local db already created on table "users_wallets".
I'd expect that all wallet dbs are missing these 3 indexes on action_queue table and therefore it should be safe to rename it.

Reply:
I see. So, there is a bug where we attempt to create different indices with the same name, right?

Reply:
or rather, several bugs

Reply:
Yes, this happened several times in different scripts. This PR resolves all such create index collisions I found.

Reply:
Yes, this happened several times in different scripts. This PR covers and fixes all such create index collisions I got aware of.